examples/misc/packed_bubbles.py

```diff
@@ -0,0 +1,158 @@
+"""
+Create a packed bubble / non overlapping bubble chart to represent scalar data.
```

Comment:
```suggestion
===================
Packed bubble chart
===================

Create a packed bubble / non overlapping bubble chart to represent scalar data.
```

Examples need a heading. This is what caused the sphinx build to fail.